app/policies/scorecard_policy.rb

```diff
@@ -33,7 +33,7 @@ def update?
   end
 
   def destroy?
-    update? && !%w(running submitted).include?(record.progress)
+    create?
```

Comment:
@hengsokly please allow to deleted only a planning & completed but not the progress otherwise the app user might encountered error during the uploading, what do you think?